bin/env_variables_all.txt

```diff
@@ -53,4 +53,13 @@ IDAM_KEY_DM_STORE=AAAAAAAAAAAAAAAA
 IDAM_KEY_FPL_CASE_SERVICE=AABBCCDDEEFFGGHH
 OAUTH2_CLIENT_CCD_ADMIN=IIIIIIIIIIIIIIII
 APPINSIGHTS_INSTRUMENTATIONKEY=SomeRandomStringForLocalDocker
-DEFINITION_STORE_DB_USE_SSL=false
\ No newline at end of file
+DEFINITION_STORE_DB_USE_SSL=false
+CCD_CASEWORKER_AUTOTEST_SOLICITOR_ONE_PASSWORD=Pa55word11
+CCD_CASEWORKER_AUTOTEST_SOLICITOR_TWO_PASSWORD=Pa55word11
+CCD_CASEWORKER_AUTOTEST_SOLICITOR_THREE_PASSWORD=Pa55word11
+CCD_CASEWORKER_ONE_AUTOTEST_PASSWORD=Pa55word11
+CCD_CASEWORKER_TWO_AUTOTEST_PASSWORD=Pa55word11
+CCD_CASEWORKER_THREE_AUTOTEST_PASSWORD=Pa55word11
+CCD_CITIZEN_TWO_AUTOTEST_PASSWORD=Pa55word11
+CCD_CITIZEN_THREE_AUTOTEST_PASSWORD=Pa55word11
+CCD_AUTOTEST_THREE_SOLICITOR_PASSWORD=Pa55word11
```

Comment:
Could we pls update these names as well, as per the latest comment on data store PR?